.github/workflows/backup-production.yml

```diff
@@ -0,0 +1,16 @@
+name: production
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  backup:
+    name: backup-firestore
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          project_id: cusec-2021-production
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+      - run: gcloud firestore export gs://cusec-2021-production-backup
```

Comment:
Does this command run for the whole time that it is backing up? (Action minutes!)
Or does it "launch" the task and the GH Action ends? (If it does it this way, do we get notified if the backup failed for whatever reason? Is that a concern?)

Reply:
It runs for the full duration and notifies if there's an error! If it ever becomes a problem there's also an `--async` flag we could use https://firebase.google.com/docs/firestore/manage-data/export-import#export_all_documents

Also I just realized, but GitHub Actions is completely free for public repos? Unlimited minutes?!?!?11 https://github.com/features/actions#pricing-details